src/lib/containers/QueryWrapper.tsx

```diff
@@ -168,6 +171,7 @@ export default class QueryWrapper extends React.Component<
       isAllFilterSelectedForFacet: {},
       loadNowStarted: false,
       lastQueryRequest: cloneDeep(this.props.initQueryRequest!),
+      isFaceted: true,
```

Comment:
components initially assume facets are available and sql results will align with facets